bci_main.py

```diff
@@ -104,6 +100,13 @@ def execute_task(task_type: dict, parameters: dict, save_folder: str) -> bool:
         language_model = None
         filename = None
 
+    # Initialize Display Window
+    # We have to wait until after the prompt for the prompt to load the signal
+    # model before displaying the window, otherwise in fullscreen mode this 
+    # throws an error.
+    display = init_display_window(parameters)
+    print_message(display, "Initializing...")
+
```

Comment:
@sci-tab , you were right about this one. I added a comment to document why display has to go here in case this gets refactored later. The new language model takes a long time to load so it may be worth exploring some other options.